python/lsst/obs/base/yamlCamera.py

```diff
@@ -166,6 +166,10 @@ def makeAmplifierList(ccd):
         amplifier.setGain(amp['gain'])
         amplifier.setReadNoise(amp['readNoise'])
         amplifier.setSaturation(amp['saturation'])
+        if 'suspect' in amp:
+            amplifier.setSuspectLevel(amp['suspect'])
+        else:
+            amplifier.setSuspectLevel(float('nan'))
```

Comment:
This is a stylistic thing... I personally prefer `np.nan` to `float('nan')`, but both are used in the codebase.  It seems that `np.nan` is slightly more prevalent if that matters.

Reply:
And this could be a bit more compact.  If `amp` is a regular dict or dict-like thing, you can do `amplifier.setSuspectLevel(amp.get('suspect', np.nan))`.  Though maybe this is unnecessarily fancy.

Reply:
I never remember the short versions of things in python until my third or fourth look.  Short form adopted.